mobile/lib/shared/models/asset.dart

```diff
@@ -504,3 +513,12 @@ extension AssetsHelper on IsarCollection<Asset> {
     return where().anyOf(ids, (q, String e) => q.localIdEqualTo(e));
   }
 }
+
+List<byte>? _decodeThumbhash(String? hash) {
+  if (hash == null) {
+    return null;
+  }
+  final hashBytes = base64.decode(base64.normalize(hash));
+  final rgbaImage = thumbhash.thumbHashToRGBA(hashBytes);
```

Comment:
Sorry, maybe my previous comment was unclear.
This will now store the actual image in the database - for large collections this will blow up the database (it has a max size of 100s MB).
I only meant to decode the base64 string to bytes (so only use the first line of this function). The thumb hash calculation to make an image should still be performed when rendering.

Reply:
My bad. Should be good now. Thank you :)